src/eos-autoupdater.c

```diff
@@ -66,6 +66,7 @@ static const char *UPDATE_STAMP_NAME = "eos-updater-stamp";
 static const char *CONFIG_FILE_PATH = SYSCONFDIR "/" PACKAGE "/eos-autoupdater.conf";
 static const char *OLD_CONFIG_FILE_PATH = SYSCONFDIR "/eos-updater.conf";
 static const char *STATIC_CONFIG_FILE_PATH = PKGDATADIR "/eos-autoupdater.conf";
+static const char *VAR_CONFIG_FILE_PATH = LOCALSTATEDIR "/local/" PACKAGE "/eos-autoupdater.conf";
```

Comment:
@ramcq I would still prefer not to depend on OSTree implementation detail here, take the most straightforward implementation and just use the exact same file path in the image builder and on the client side; but at the end of the day it's bikeshedding and don't feel like arguing it.
I will merge the PR using this path for now.